socorro/processor/mozilla_transform_rules.py

```diff
@@ -676,7 +676,7 @@ def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
         processed_crash.flash_version = ''
         flash_version = None
         for index, a_module in enumerate(
-            processed_crash['json_dump']['modules']
+            processed_crash['json_dump'].get('modules', [])
```

Comment:
This still assumes that you get back a list. We should pull this out and make sure it's a list and not a string which you can also enumerate over, but would produce junk data later.